### PR TITLE
Fix namespace registration fee for account creation

### PR DESCRIPTION
**Pull request by Buckram123**
### Checklist

- [x] CI is green.
- [x] Changelog updated.

### CODE REVIEW

**Review by CyberHoward — CHANGES_REQUESTED** [review]
I think interchain account creation should be free. Once you have an initial account somewhere you should be able to create interchain accounts for that account without any payment of sorts.

**Comment by Buckram123** [comment]
> I think interchain account creation should be free. Once you have an initial account somewhere you should be able to create interchain accounts for that account without any payment of sorts.

This PR only impacts namespace claiming fee, not account creation

**Review by CyberHoward — APPROVED** [review]
Awesome catch and fix!! :)

File: framework/contracts/native/account-factory/tests/create_account.rs [hunk 1]
```diff
@@ -325,6 +332,76 @@ fn create_one_account_with_namespace() -> AResult {
...
+        Some(namespace_to_claim.to_string()),
+        // Account creation fee not covered
+        &[],
+    );
+    assert!(err
+        .unwrap_err()
+        // Error type is inside contract, not the package
+        .root()
+        .to_string()
+        .contains("Invalid fee payment sent."));
```
Comment (Buckram123) [inline framework/contracts/native/account-factory/tests/create_account.rs:390]: @CyberHoward wdyt do we want to move error types to packages as well?

Comment (CyberHoward) [reply framework/contracts/native/account-factory/tests/create_account.rs:390]: Error types of the contract? I'd prefer them to stay in the contract.

**Pull request merged**